DWAPlannerROS:
  # Velocity limits
  max_vel_x: 0.5         # Forward max speed
  min_vel_x: 0.0       # Enable backward motion

  max_vel_theta: 1.0     # Fast turning in place
  min_vel_theta: -1.0

  acc_lim_x: 1.2        # Fast acceleration/deceleration
  acc_lim_theta: 2.0
  acc_lim_y: 0.0

  # In-place rotation enabled here
  min_in_place_vel_theta: 0.5   # Minimum speed to allow in-place turning

  # Trajectory simulation
  sim_time: 2.0
  sim_granularity: 0.025
  vx_samples: 10
  vtheta_samples: 20
  controller_frequency: 10.0

  # Planner bias tuning
  path_distance_bias: 32.0      # Follow global path
  goal_distance_bias: 24.0      # Move toward goal
<<<<<<< HEAD
  occdist_scale: 0.02           # Avoid obstacles

  forward_point_distance: 0.3
=======
  occdist_scale: 0.15           # Avoid obstacles

  forward_point_distance: 0.325
>>>>>>> 77d0c76948a66632b3bf7148c1832289837f4468
  stop_time_buffer: 0.2
  scaling_speed: 0.4
  max_scaling_factor: 0.3

  # Goal tolerances
<<<<<<< HEAD
  yaw_goal_tolerance: 0.3
  xy_goal_tolerance: 0.2
=======
  yaw_goal_tolerance: 0.25
  xy_goal_tolerance: 0.1
>>>>>>> 77d0c76948a66632b3bf7148c1832289837f4468
  latch_xy_goal_tolerance: true

  # Behavior
  prune_plan: true
  restore_defaults: false
  publish_cost_grid: false
<<<<<<< HEAD
  global_frame_id: map

=======
  global_frame_id: odom
>>>>>>> 77d0c76948a66632b3bf7148c1832289837f4468
